spec/unit/resource_controller/data_access_spec.rb

```diff
@@ -88,4 +88,48 @@
       end
     end
   end
-end
+
+  describe "find_collection" do
+    let(:appliers) do
+      ActiveAdmin::ResourceController::DataAccess::COLLECTION_APPLIES
+    end
+    let(:scoped_collection) do
+      double "ScopedCollectionChain"
+    end
+    before do
+      allow(controller).to receive(:scoped_collection).and_return(scoped_collection)
+    end
+
+    it "should return chain with all appliers " do
+      appliers.each do |applier|
+        expect(controller).to receive("apply_#{applier}").
+          with(scoped_collection).
+          once.
+          and_return(scoped_collection)
+      end
+      expect(controller).to receive(:collection_applies).with({}).and_call_original.once
+      controller.send :find_collection
+    end
+
+    describe 'collection_applies' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.